src/sentry/static/sentry/app/views/issueList/actions.tsx

```diff
@@ -810,6 +808,21 @@ const IconPad = styled('span')`
   top: ${space(0.25)};
 `;
 
+const SelectAllNotice = styled('div')`
+  background-color: #fffdf7;
+  border: 1px solid #e7d796;
+  text-align: center;
+  margin: 0 -1px -1px;
+  padding: ${space(0.5)} ${space(1.5)};
+  z-index: 2;
+`;
+
+const SelectAllLink = styled('a')`
+  margin-left: ${space(1)};
+  font-size: ${p => p.theme.fontSizeLarge};
+  color: ${p => p.theme.blue400};
```

Comment:
Do we need this? We should use default link color imo

I think it'd be preferable to use `<Button[priority="link"]>` if possible, but I know there can by styling challenges with that. (This should not block PR)

Reply:
worked with robin on the colors